### ISSUE TITLE
Wrong condition in Chooser.detach()

### ISSUE THREAD

**Issue by snake-scaly** [snake-scaly/usher-bot#1]
The condition
```
if (_client) return;
```
makes no sense there. It should be reversed.
